### PR TITLE
Fix domain issue with error message

### PR DESCRIPTION
**Pull request by oshtaier**
Fix issue with locator for error message
Bonus: Some refactoring according to our code style

```
nosetests tests/foreman/ui/test_domain.py
..........S
----------------------------------------------------------------------
Ran 11 tests in 515.690s

OK (SKIP=1)
```

### CODE REVIEW

**Comment by sghai** [comment]
ACK

File: tests/foreman/ui/test_domain.py [hunk 1]
```diff
@@ -197,19 +191,15 @@ def test_positive_set_parameter_long(self):
...
         with Session(self.browser) as session:
             make_domain(session, name=domain_name, description=description)
-            element = self.domain.search(description)
-            self.assertIsNotNone(element)
-            try:
-                self.domain.set_domain_parameter(
-                    description, param_name, param_value)
-            except UIError as err:
-                self.fail(err)
+            self.assertIsNotNone(self.domain.search(description))
```
Comment (omaciel) [inline tests/foreman/ui/test_domain.py]: 👍

File: tests/foreman/ui/test_domain.py [hunk 2]
```diff
@@ -222,19 +212,15 @@ def test_positive_set_parameter_blank(self):
...
```
Comment (omaciel) [inline tests/foreman/ui/test_domain.py]: 👍

File: tests/foreman/ui/test_domain.py [hunk 3]
```diff
@@ -243,57 +229,45 @@ def test_negative_set_parameter(self):
...
-            element = self.domain.search(description)
-            self.assertIsNotNone(element)
-            try:
-                self.domain.set_domain_parameter(
-                    description, param_name, param_value)
-            except UIError as err:
-                self.fail(err)
-            self.assertIsNotNone(session.nav.wait_until_element(
-                common_locators['common_param_error']))
+            self.assertIsNotNone(self.domain.search(description))
```
Comment (omaciel) [inline tests/foreman/ui/test_domain.py]: 👍

File: tests/foreman/ui/test_domain.py [hunk 4]
```diff
@@ -243,57 +229,45 @@ def test_negative_set_parameter(self):
...
+            self.domain.set_domain_parameter(
+                description,
+                param_name=gen_string('alpha', 256),
+                param_value=gen_string('alpha', 256),
+            )
+            self.assertIsNotNone(self.domain.wait_until_element(
```
Comment (omaciel) [inline tests/foreman/ui/test_domain.py]: 👍

File: tests/foreman/ui/test_domain.py [hunk 5]
```diff
@@ -243,57 +229,45 @@ def test_negative_set_parameter(self):
...
     @tier2
     def test_negative_set_parameter_same(self):
         """Again set the same parameter for domain with name and value.
 
         @id: 6266f12e-cf94-4564-ba26-b467ced2737f
 
-        @Assert: Domain parameter is not updated.
-
-        @BZ: 1123360
+        @Assert: Domain parameter with same values is not created.
```
Comment (omaciel) [inline tests/foreman/ui/test_domain.py]: 💯

**Pull request merged**